### ISSUE TITLE
Hide/show vkb depending on ContextKit keyboard opened state

### ISSUE THREAD

**Issue by kimmoli** [nemomobile/fingerterm#40]
preparations for tohkbd2.

tohkbd daemon will emit a dbus signal on `systembus` when it is connected or disconnect.
For optimum user experience with fingerterm, the "ui/vkbShowMethod" should be set to "off" when tohkbd is connected, and to previous value when disconnected.

`path=/; interface=com.kimmoli.tohkbd2; member=keyboardConnectedChanged`
`boolean false` when disconnected
and
`boolean true` when connected

**Comment by thp** [comment]
Agree that the way to do it would be to listen to a contextkit/statefs property, here's the keyboard provider source:

https://github.com/nemomobile/statefs-providers/blob/master/src/keyboard_generic/provider_keyboard_generic.cpp

Added an issue there: https://github.com/nemomobile/statefs-providers/issues/28

**Comment by deztructor** [comment]
iirc @spiiroin is going to provide some signal to reflect "Opened" state to be used e.g. by statefs, isn't it?

**Comment by spiiroin** [comment]
@deztructor: yes, https://github.com/nemomobile/mce/pull/230 (in review) adds the following signals

/usr/lib/qt5/bin/qdbus --system com.nokia.mce /com/nokia/mce/signal|grep keyb
signal void com.nokia.mce.signal.keyboard_available_state_ind(QString keyboard_state)
signal void com.nokia.mce.signal.keyboard_slide_state_ind(QString slide_state)

The vkb related statefs could be derived from the "available" data.

**Comment by kimmoli** [comment]
nemomobile/statefs-providers#41 is required for correct operation, at least using 0.2.74 with this  nemomobile-packages/maliit-framework#8 seems to work(ish)
